BookOfHabits/BookOfHabits/Requests/Person/CreatePersonRequest.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace BookOfHabits.Requests.Person
+{
+    public class CreatePersonRequest
+    {
+        public Guid Id { get; init; }
+        public required string Name { get; init; }
```

Comment:
add [Require] attribute, where it need

Reply:
This is a validation attribute. I don't like using attributes, I use Fluentvalidation